backend/src/main/java/edu/ncsu/csc540/s23/backend/constants/SetupQuery.java

```diff
@@ -38,15 +38,15 @@ public class SetupQuery {
     public static final String CREATE_TABLE_ARTIST = "CREATE TABLE IF NOT EXISTS Artist (user_id int,rlabel_id int NOT NULL,primary_genre_id int,status varchar(16) NOT NULL,`type` varchar(16) NOT NULL,artist_country varchar(64),PRIMARY KEY(user_id),FOREIGN KEY (`user_id`) REFERENCES `User` (`user_id`) ON DELETE CASCADE,FOREIGN KEY (`rlabel_id`) REFERENCES `Record_Label` (`rlabel_id`) ON DELETE CASCADE,FOREIGN KEY (`primary_genre_id`) REFERENCES `Genre` (`genre_id`) ON DELETE SET NULL);";
     public static final String CREATE_TABLE_PODCAST_HOST = "CREATE TABLE IF NOT EXISTS Podcast_Host (user_id int,city varchar(64),PRIMARY KEY(user_id),FOREIGN KEY (`user_id`) REFERENCES `User` (`user_id`) ON DELETE CASCADE);";
     public static final String CREATE_TABLE_ALBUM = "CREATE TABLE IF NOT EXISTS Album (album_id int NOT NULL AUTO_INCREMENT,album_name varchar(64) NOT NULL,release_year int NOT NULL,edition varchar(30) NOT NULL,PRIMARY KEY (album_id));";
-    public static final String CREATE_TABLE_SONG = "CREATE TABLE IF NOT EXISTS Song (song_id int NOT NULL AUTO_INCREMENT,album_id int,title varchar(64) NOT NULL,duration time NOT NULL,track_no int NOT NULL,release_date DATETIME NOT NULL,release_country varchar(64) NOT NULL,language varchar(30) NOT NULL,royalty_rate DECIMAL(10,2) NOT NULL,PRIMARY KEY (song_id, album_id),UNIQUE(album_id, title),FOREIGN KEY (`album_id`) REFERENCES `Album` (`album_id`) ON DELETE CASCADE);";
-    public static final String CREATE_TABLE_PODCAST = "CREATE TABLE IF NOT EXISTS Podcast (podcast_id int NOT NULL AUTO_INCREMENT,pname varchar(64) NOT NULL,planguage varchar(30) NOT NULL,country varchar(64) NOT NULL,PRIMARY KEY(podcast_id));";
+    public static final String CREATE_TABLE_SONG = "CREATE TABLE IF NOT EXISTS Song (song_id int NOT NULL AUTO_INCREMENT,album_id int,title varchar(64) NOT NULL,duration time NOT NULL,track_no int NOT NULL,release_date DATETIME NOT NULL,release_country varchar(64) NOT NULL,language varchar(30) NOT NULL,royalty_rate DECIMAL NOT NULL,PRIMARY KEY (song_id, album_id),UNIQUE(album_id, title),FOREIGN KEY (`album_id`) REFERENCES `Album` (`album_id`) ON DELETE CASCADE);";
+    public static final String CREATE_TABLE_PODCAST = "CREATE TABLE IF NOT EXISTS Podcast (podcast_id int NOT NULL AUTO_INCREMENT,user_id int, pname varchar(64) NOT NULL,planguage varchar(30) NOT NULL,country varchar(64) NOT NULL,PRIMARY KEY(podcast_id), FOREIGN KEY (`user_id`) REFERENCES `Podcast_Host` (`user_id`) ON DELETE CASCADE);";
```

Comment:
ON DELETE CASCADE or NULL? Do we need to delete a podcast if the podcast host gets deleted or can we just mark podcast host as null for the podcast